tutorials/navigation/navigation_using_navigationpathqueryobjects.rst

```diff
@@ -70,3 +99,197 @@ has a large quantity of simultaneous agents that regularly update their paths.
+In order to improve the quality of paths returned by the query various ``path_postprocessing`` options exist.
+
+- The ``PATH_POSTPROCESSING_CORRIDORFUNNEL`` post processing shortens paths by funneling paths around corners **inside the available polygon corridor**.
+  
+  This is the default post processing and usually also the most useful as it gives the shortest path result **inside the available polygon corridor**. If the polygon corridor is already suboptimal, e.g. due to a suboptimal navigation mesh layout, the funnel can snap to unexpected polygon corners causing detours.
+
+- The ``PATH_POSTPROCESSING_EDGECENTERED`` post processing forces all path points to be placed in the middle of the crossed polygon edges  **inside the available polygon corridor**.
+  
+  This post processing is usually only useful when used with strictly tile-like navigation mesh polygons that are all evenly sized and where the excepted path following is also constrained to cell centers, e.g. typical grid game with movement constrained to grid cell centers.
+
+- The ``PATH_POSTPROCESSING_NONE`` post processing returns the path as is how the pathfinding traveled **inside the available polygon corridor**.
+  
+  This post processing is very useful for debug as it shows how the path search traveled from closest edge point to closet edge point and what polygons it picked.
+  A lot of unexpected or suboptimal path results can be immediately explained by looking at this raw path and polygon corridor.
+
+Path simplification
+-------------------
+
+.. tip::
+
+    Path simplification can help steering agents or agents that jitter on thin polygon edges.
+
+.. figure:: img/path_simplification_diff.webp
+   :align: center
+   :alt: Path point difference with or without path simplification
+
+   Path point difference with or without path simplification.
+
+If ``simplify_path`` is enabled a variant of the Ramer-Douglas-Peucker path simplification algorithm is applied to the path.
+This algorithm straightens paths by removing less relevant path points depending on the used ``simplify_epsilon``.
+
+Path simplification helps with all kinds of agent movement problems in "open fields" that are caused by having many unnecessary polygon edges.
+E.g. a terrain mesh when baked to a navigation mesh can cause an excessive polygon count due to all the small (but for pathfinding almost meaningless) height variations in the terrain.
+
+Path simplification also helps with "steering" agents because they only have more critical corner path points to aim for.
+
+.. Warning::
+
+    Path simplification is an additional final post-processing of the path. It adds extra performance cost the query so only enabled the simplification when actually needed.
+
+.. note::
+
+    Path simplification is exposed on the NavigationServer as a generic function. It can be used outside of navigation queries for all kinds of position holding arrays as well.
```

Comment:
```suggestion
    Path simplification is exposed on the NavigationServer as a generic function. It can be used outside of navigation queries for all kinds of position arrays as well.
```